src/nn/wasm/src/bind/src/binding.cpp

```diff
@@ -153,12 +168,12 @@ namespace binding_utils {
                           intptr_t outputData,
                           const RuntimeShape& im2colShape, 
                           intptr_t im2colData) {
-    optimized_ops::Conv(op_params, 
-                        inputShape, (const float*)inputData, 
-                        filterShape, (const float*)filterData, 
-                        biasShape, (const float*)biasData, 
-                        outputShape, (float*)outputData, 
-                        im2colShape, (float*)im2colData);
+    optimized_ops::Conv(op_params, inputShape,
```

Comment:
Why not call `binding_utils::get_cpu_flags(&cpu_backend_context, &cpu_flags);` as `DepthwiseConv`.

Reply:
Because get_cpu_flags modify the value of cpu_flags based on the value of cpu_backend_context. As cpu_flags isn't an input parameter of Conv, so I think there is no need to call function get_cpu_flags.

Reply:
`cpu_flags` may impact the execution of Conv. I suppose you need to call it before each op computation. You may need to refer to tflite code for the usage and report back here. Thanks.

Reply:
I have checked the source code.Since cpu_flags is generated entirely based on cpu_backend_context, the function can update cpu_flags inside if needed. And the struct CpuFlags only appears in the implementation of DepthwiseConv, so I think there is no need to call it before each op computation.

Reply:
Sounds good.